charts/datahub/Chart.yaml

```diff
@@ -4,40 +4,35 @@ description: A Helm chart for LinkedIn DataHub
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.2.109
+version: 0.2.110
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
 appVersion: 0.9.0
 dependencies:
   - name: datahub-gms
-    version: 0.2.11
+    version: 0.2.110
```

Comment:
Is the idea that we bump all sub-charts when an update to a single change is made regardless of where it is as long as it bumps the overall chart version?

Reply:
I think that it does make sense to version them the same. These charts are tightly coupled, not necessarily in the helm chart perspective however definitely the application. I think that other projects would actually not have broken the charts into subcharts since it is unlikely that someone would actually deploy them independently. In the future, if there is ever a shared config map in the parent chart or other resource between them it would make even more sense that they are versioned consistently.

Reply:
@pedro93 yes
@david-leifker exactly, I would like to get rid of sub-charts all together in the long run to simplify our setup.